compiler_opt/tools/regalloc_trace/compile_corpus.py

```diff
@@ -92,9 +89,15 @@ def main(_) -> None:
   if _MODE.value == 'bc':
     for module in train_corpus.module_specs:
       command_file = os.path.join(_CORPUS_PATH.value, module.name) + '.cmd'
-      new_command_file_path = os.path.join(_OUTPUT_PATH.value,
-                                           module.name) + '.cmd'
-      shutil.copy(command_file, new_command_file_path)
+      with open(command_file, encoding='utf-8') as command_file_handle:
+        command_tuple = tuple(command_file_handle.read().split('\0')[:-1])
+      command_tuple = command_tuple + ('-disable-llvm-passes',)
+      new_command_file_path = (
```

Comment:
Wait, so compiling to "bc" means... what. Taking the pre-opt IR and leaving it as-is?

Reply:
Taking the pre-opt IR and then running the rest of the middle end pipeline. We don't want to run any middle end passes after that when we compile to asm, so we add the -disable-llvm-passes flag to prevent them from running.